documentation/error-handling.md

```diff
@@ -0,0 +1,64 @@
+# Error handling
+
+UI extensions support standard web errors. You can add `unhandledrejection` listener for unhandled promise rejections or `error` listener for other exceptions like Javascript runtime error or a resource fails to load.
+
+```javascript
+// For unhandled promise rejections
+self.addEventListener('unhandledrejection', (error) => {
+  console.warn('event unhandledrejection', error);
+});
+
+// For other exceptions
+self.addEventListener('error', (error) => {
+  console.warn('event error', error);
+});
+```
+
+## Reporting errors
+
+It's possible to use 3rd party libaries for reporting errors like Bugsnag or Sentry. However, it might require some extra configs or setup depending on which libraries you are using because UI extensions are run inside WebWorker.
```

Comment:
```suggestion
It's possible to use 3rd party libaries like Bugsnag or Sentry for reporting errors. However, it might require extra configuration or setup, depending on which libraries you use, because UI extensions run inside [WebWorkers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API).
```